src/components/Answers/Answers.tsx

```diff
@@ -3,22 +3,22 @@
 import { h } from 'preact';
 import cx from 'classnames';
 import Template from '../Template/Template';
-import { AnswersTemplates } from '../../widgets/answers/answers';
+import {
+  AnswersCSSClasses,
+  AnswersTemplates,
+} from '../../widgets/answers/answers';
 import { Hits } from '../../types';
 
-type AnswersCSSClasses = {
-  root: string;
-  emptyRoot: string;
-  header: string;
-  loader: string;
-  list: string;
-  item: string;
-};
+export type AnswerComponentCSSClasses = Required<
+  {
+    [TClassName in keyof AnswersCSSClasses]: string;
+  }
+>;
```

Comment:
maybe a helper for this in /types? 

```ts
type ComponentCSSClasses<TCSSClasses> = Required<{ [className in keyof TCSSClasses]: string; }>;
```

Reply:
Added here, looks even cleaner! https://github.com/algolia/instantsearch.js/pull/4774/commits/58fc6225cdd9fe01f5521e1ec7d80d36efb7d32f